app/code/core/Mage/Payment/Model/Resource/Restriction/Collection.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/**
+ * Maho
+ *
+ * @package    Mage_Payment
+ * @copyright  Copyright (c) 2025 Maho (https://mahocommerce.com)
+ * @license    https://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+
+class Mage_Payment_Model_Resource_Restriction_Collection extends Mage_Core_Model_Resource_Db_Collection_Abstract
+{
+    #[\Override]
+    protected function _construct(): void
+    {
+        $this->_init('payment/restriction');
+    }
+
+    public function addActiveFilter(): self
+    {
+        return $this->addFieldToFilter('status', Mage_Payment_Model_Restriction::STATUS_ENABLED);
+    }
+
+    public function addPaymentMethodFilter(string $paymentMethod): self
+    {
+        return $this->addFieldToFilter([
+            ['attribute' => 'payment_methods', 'like' => '%' . $paymentMethod . '%'],
```

Comment:
Is there a potential SQL injection here? Although I can't find any usage of this method, since `git grep addPaymentMethodFilter` only returns this definition.